src/components/macroview/rightPanel/editForms/SystemEventActionForm.tsx

```diff
@@ -9,11 +9,19 @@ interface Props {
   selectedElementId: number
 }
 
-export default function SystemEventActionForm({ selectedElement, selectedElementId }: Props) {
-  const SelectedElementFormComponent = useMemo(() => {
+export default function SystemEventActionForm({
+  selectedElement,
+  selectedElementId
+}: Props) {
+  return useMemo(() => {
```

Comment:
I guess you didn't add this useMemo, but it's kinda weird to be here. If this needs to be memoised, it should be a memoised component, not using useMemo
ref: https://react.dev/reference/react/memo

Reply:
I tried to remove it and didnt see any negative outcome: [1f7fa2f](https://github.com/WootingKb/wooting-macros/pull/194/commits/1f7fa2fa3e2c674d965a85d3dfd5f231ec9e3119)